templates/base/head_navbar.tmpl

```diff
@@ -3,15 +3,15 @@
 	{{$notificationUnreadCount = call .NotificationUnreadCount}}
 {{end}}
 
-<nav id="navbar" aria-label="{{ctx.Locale.Tr "aria.navbar"}}">
+<nav id="navbar" class="ui secondary menu" aria-label="{{ctx.Locale.Tr "aria.navbar"}}">
 	<div class="navbar-left">
```

Comment:
I think `ui secondary menu` should be on `class="navbar-left"`?

Ideally the layout should be:

```
<div "ui secondary menu">
   <div "item active">...</>
   <div "item">...</>
</>
```

Reply:
I don't think so. I believe the “notifications” on the right should also take effect. It's just that “PageIsNotifications” is not set in Go at the moment.

Reply:
> Actually there could be another choice: totally drop Fomantic UI, and add a `.navbar-left > .item.active, .navbar-right > .item.active { ... }` style

Hmm ...... I think we need to do this, because the styles are all done by us, not by Fomantic UI, and there is a plan to remove Fomantic UI step by step.

![image](https://github.com/user-attachments/assets/694d7e09-1eab-46b1-86b1-bc19873e5beb)